specs-go/config.go

```diff
@@ -90,8 +98,10 @@ type Mount struct {
 	// Source specifies the source path of the mount.  In the case of bind mounts on
 	// Linux based systems this would be the file on the host.
 	Source string `json:"source"`
-	// Options are fstab style mount options.
-	Options []string `json:"options,omitempty"`
+	// Options are fstab style mount options. (this field is platform dependent)
+	Options []string `json:"options,omitempty" platform:"linux,solaris"`
```

Comment:
I'm not sure - that PR is documentation. The `Options` are very different between *nix and Windows - Windows obviously has no concept of fstab. I added the `platform` tag to make this field not applicable to Windows, and added a separate field which is for Windows.